### PR TITLE
stats, metrics: add and fix stats related grafana panel

### PR DESCRIPTION
**Pull request by winoros**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### What changed and how does it work?

Add two new panels for stats delta loading and updating.
Fix the y-axe of the auto analyze and the sync load.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by winoros** [comment]
image
image
Y-axe change

**Comment by winoros** [comment]
image
The newly added two.

File: pkg/metrics/stats.go [hunk 1]
```diff
@@ -134,4 +137,28 @@ func InitStatsMetrics() {
...
+	StatsDeltaUpdateHistogram = NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "tidb",
+			Subsystem: "statistics",
+			Name:      "stats_delta_update_duration_seconds",
+			Help:      "Bucketed histogram of processing time for the background stats_meta update job",
+			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 24), // 10ms ~ 24h
+		},
+	)
+	StatsLoadObjectGauge = NewGaugeVec(prometheus.GaugeOpts{
```
Comment (AilinKid) [inline pkg/metrics/stats.go:158]: any usage of this one

Comment (winoros) [reply pkg/metrics/stats.go]: Just a try and found that it's very well. deleted.

**Review by hawkingrei — APPROVED** [review]

**Review by AilinKid — APPROVED** [review]

**Pull request merged**